db/c.cc

```diff
@@ -2154,6 +2154,10 @@ void rocksdb_options_destroy(rocksdb_options_t* options) {
   delete options;
 }
 
+rocksdb_options_t* rocksdb_options_copy(rocksdb_options_t* options) {
```

Comment:
I prefer it to be called `rocksdb_options_create_copy` to be extra clear about allocation.